tasks/Qc.wdl

```diff
@@ -185,6 +185,10 @@ task CollectRawWgsMetrics {
       OUTPUT=~{metrics_filename} \
       USE_FAST_ALGORITHM=true \
       READ_LENGTH=~{read_length}
+
+    sed 's/picard.analysis.WgsMetrics/picard.analysis.CollectWgsMetrics\$WgsMetrics/' ~{metrics_filename}  > ~{metrics_filename}.fixed
+
+    mv ~{metrics_filename}.fixed ~{metrics_filename}
```

Comment:
Could use `sed`'s `-i` option to avoid this step if you wanted.

Reply:
thanks! that was one reason for making the PR, I figured there was a better way.